src/modules/identityId.ts

```diff
@@ -0,0 +1,21 @@
+import { AuthAPI, AuthIdentityPayload } from './auth-api'
+import { AuthIdentity } from 'decentraland-crypto-fetch'
+
+export async function getIdentityId(identity: AuthIdentity): Promise<string> {
+  if (!identity) {
```

Comment:
The function signature does not allow identity to be undefined, would you mind removing this check?